src/Compilers/CSharp/Portable/Symbols/Synthesized/RefKindVector.cs

```diff
@@ -69,5 +76,57 @@ public override int GetHashCode()
         {
             return _bits.GetHashCode();
         }
+
+        public string ToRefKindString()
+        {
+            var pooledBuilder = PooledStringBuilder.GetInstance();
+            var builder = pooledBuilder.Builder;
+
+            builder.Append("{");
+
+            int i = 0;
+            foreach (int byRefIndex in this.Words())
+            {
+                if (i > 0)
+                {
+                    builder.Append(",");
+                }
+
+                builder.AppendFormat("{0:x8}", byRefIndex);
+                i++;
+            }
+
+            builder.Append("}");
+            Debug.Assert(i > 0);
+
+            return pooledBuilder.ToStringAndFree();
+        }
+
+        public static RefKindVector Parse(string refKindString, int capacity)
+        {
+            int index = 0;
+            // To avoid having to map ref kinds again we can just deserialize the BitVector which is what
+            // would have produced the string in the method above
+            var bitVector = BitVector.Create(capacity * 2);
+            foreach (var word in refKindString.Split(','))
```

Comment:
>refKindString [](http://example.com/codeflow?start=33&length=13)

BitVector represents bits as an array of `uint`s. So it seems that you could just create BitVector directly from `uint`s